src/lamp_py/bus_performance_manager/write_events.py

```diff
@@ -10,14 +12,16 @@
 from lamp_py.aws.s3 import upload_file
 
 
-def write_bus_metrics() -> None:
+def write_bus_metrics(
```

Comment:
The core of this change seems to calling `bus_performance_metrics` with a user defined date range and then writing to a local path. 

This could be accomplished more easily with a new function that doesn't add such complexity to existing functions meant to be run by the ECS.

`event_files_to_load` and `write_bus_metrics` are designed to provide a specific scope of functionality to the ECS application. They are scaffolding around a call of `bus_performance_metrics` to allow the ECS to produce the expected output of parquet files. 

Could you re-write the changes as a new `local_write_bus_metrics` function to maintain a separation of concerns?